### ISSUE TITLE
npm install should not be called from package.json postinstall

### ISSUE THREAD

**Issue by netik** [nhn/tui.image-editor#286]
## Version
3.4.0 (problem is present in current code as well)

## Development Environment
Google Chome, latest

## Current Behavior

In the `postinstall` stanza, the image editor attempts to sneakily install fabric via npm:
```
 "postinstall": "npm install --no-save --no-optional fabric@^3.3.2"
```

This breaks projects using `yarn`. If this package is a dependency, then it should be listed as such and not installed as part of postinstall. If it has to be in postinstal, then you should figure out if the user is running `yarn` or `npm` and run the right package manager. This behaviour breaks yarn.

## Expected Behavior

Clean package install with proper dependency management.

**Comment by jinwoo-kim-nhn** [comment]
Previously, if you did not install `--no-optional` in `fabric` version 1.6.7, you ran into problems.

The fabric version I currently use has been updated to 3.x.x so there seems to be no problem installing on the dependencies.

Thank you for your feedback and we will test it a bit more and fix it.

**Comment by jinwoo-kim-nhn** [comment]
It was released in v3.7.3.
